pkg/otlp/internal/serializerexporter/exporter.go

```diff
@@ -176,8 +177,30 @@ func (e *exporter) ConsumeMetrics(ctx context.Context, ld pmetric.Metrics) error
 	}
 
 	consumer.addTelemetryMetric(e.hostname)
+	if hasRuntimeMetrics(ld) {
+		consumer.addRuntimeTelemetryMetric(e.hostname)
+	}
 	if err := consumer.Send(e.s); err != nil {
 		return fmt.Errorf("failed to flush metrics: %w", err)
 	}
 	return nil
 }
+
+func hasRuntimeMetrics(ld pmetric.Metrics) bool {
+	rms := ld.ResourceMetrics()
+	for i := 0; i < rms.Len(); i++ {
+		rm := rms.At(i)
+		ilms := rm.ScopeMetrics()
+		for j := 0; j < ilms.Len(); j++ {
+			ilm := ilms.At(j)
+			metricsArray := ilm.Metrics()
+			for k := 0; k < metricsArray.Len(); k++ {
+				md := metricsArray.At(k)
+				if strings.Contains(md.Name(), "process.runtime") {
```

Comment:
is this same for all languages ?

Reply:
Yes, it's a requirement from semantic conventions: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md

Reply:
Yes, it's a requirement from semantic conventions: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md